src/maker/api.rs

```diff
@@ -592,9 +617,13 @@ pub fn check_for_idle_states(maker: Arc<Maker>) -> Result<(), MakerError> {
                         "[{}] Spawning recovery thread after Taker dropped",
                         maker.config.port
                     );
-                    let handle = std::thread::spawn(move || {
-                        recover_from_swap(maker_clone, outgoings, incomings).unwrap()
-                    });
+                    let handle = std::thread::Builder::new()
```

Comment:
there are some places we are using `std::thread`, in some just `thread`. I would suggest to just keep `std::thread` everywhere. 

Its not conventional to import `thread`, `io`, `file` etc. as its a tiny name space for full specification.